extensions/divina.md

```diff
@@ -150,28 +142,26 @@ This document introduces a new collection role to fulfill that goal:
 | ----- | ---------- | ------------------- | --------- |
 | `guided` | Identifies a collection containing guided navigation into a publication. | Yes  | No  |
 
-To avoid duplicating content between `readingOrder` and `guided`, Link Objects referenced in `guided` <span class="rfc">should</span> only contain `href`, `title` and `children`.
+To avoid duplicating content between `readingOrder` and `guided`, Link Objects referenced in `guided` <span class="rfc">must</span> only contain `href` and `title`.
 
 This current draft does not cover guided navigation over alternate versions of each image resource.
```

Comment:
we can now say "specification" isn't it?

Reply:
It's still a draft, we haven't covered everything planned in there yet.